lib/bedrock/index.js

```diff
@@ -1 +1,22 @@
-module.exports = (data) => ({})
+const { buildIndexFromArray } = require('./indexer')
+
+module.exports = (data) => {
+  return {
+    loadItemStates (itemStates) {
+      for (const item of itemStates) {
+        const name = item.name.replace('minecraft:', '')
+
+        if (data.itemsArray) {
```

Comment:
This seems pretty inefficient (double for loop), can you explain what you're doing here?

Reply:
So basically it loops over the item states, and for each item in there it gets the name, then change the id of the existing value with the same item name.

Reply:
why not just something like this:
```js
const items = []
for (const item of itemStates) {
  const name = item.name.replace('minecraft:', '')
  items.push({ ...staticData.itemsByName[name], id: item.runtime_id })
}
data.itemsArray = items
...
```
?

Reply:
I didn't think of that